app/models/ems_cluster.rb

```diff
@@ -232,11 +232,11 @@ def all_resource_pools_with_default
 
   # Parent relationship methods
   def parent_folder
-    detect_ancestor(:of_type => "EmsFolder") { |a| !a.is_datacenter && !["host", "vm"].include?(a.name) } # TODO: Fix this to use EmsFolder#hidden?
+    detect_ancestor(:of_type => "EmsFolder") { |a| a.class == EmsFolder && !["host", "vm"].include?(a.name) } # TODO: Fix this to use EmsFolder#hidden?
```

Comment:
I think I'd prefer `a.class != Datacenter`, because there is likely to be other folder types